l10n_ch_hr_payroll/README.rst

```diff
@@ -19,13 +24,29 @@ info@open-net.ch
 **History :**
 
 V1.0.0: 2014-11-07/Sge
-    * Add Salary rule categories
-    * Add Salary rules
-    * Add Employee children in school
-    * Add Contract LPP rate
+    * Add Salary rule categories.
+    * Add Salary rules.
+    * Add Employee children in school.
+    * Add Contract LPP rate.
 
 V1.0.1: 2014-11-11/Sge
-    * Set the 'LPP rate'' digits to 'Payroll Rate' decimal accuracy
+    * Set the 'LPP rate'' digits to 'Payroll Rate' decimal accuracy.
 
 V1.0.2:
     * Add some minor changes, based on pull request #66 comments.
+
+V1.0.3-4:
+    * Add LPP Amount to contract.
+    * Add Worked Hours Rate to contract.
+    * Add Hourly Rate to contract.
+    * Compute the Wage of contract, based on Worked Hours and Hourly Rate.
+    * Add new salay rules
+
+V1.0.5:
+    * Add Holiday Rate to contract.
+    * Update "Indemnité vacances 8,33%" rule to include "Holiday Rate" from contract.
+
+V1.0.6:
+    * Move salary.rule from CSV file to XML file.
+    * Import salary.rule only at install.
+    * Internal reorganization of files (.py and .xml).
```

Comment:
Please take the opportunity to follow OCA README template:

https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst